crates/sui-types/src/unit_tests/base_types_tests.rs

```diff
@@ -328,11 +328,9 @@ fn test_move_object_size_for_gas_metering() {
     );
     let size = object.object_size_for_gas_metering();
     let serialized = bcs::to_bytes(&object).unwrap();
-    // The result of object_size_for_gas_metering() will be smaller due to not including
-    // all the metadata data needed for serializing various types.
     // If the following assertion breaks, it's likely you have changed MoveObject's fields.
     // Make sure to adjust `object_size_for_gas_metering()` to include those changes.
-    assert_eq!(size + 3, serialized.len());
+    assert_eq!(size, serialized.len());
```

Comment:
I'm not really sure why, but the size of `MoveObject` has changed because of the addition of `shared_at_version`, and the size of the serialized representation seems not to have changed.

Reply:
Okay, I figured out what's going on here:  Part of `size` is the size of the `Owned` enum which we added the `initial_shared_version` to, so it's grown in size, however the BCS encoding of an enum is to ULEB128 encode the variant, and then BCS encode that variant's fields in sequence after it.

The owner in these tests is an address, so they aren't shared, which means their BCS variant didn't get the new field that was added to `Owner`, and overall the size did not change.